shell/README.md

```diff
@@ -5,6 +5,19 @@
 - Explain when and why command-line interfaces should be used instead of graphical interfaces.
 {% endobjectives %}
 
+{% prereq "Prerequisites" %}
+In this lesson we will use the example `data-shell` directory to provide a
+common set of files that everyone has access to. In order to create these files
+and directories, login to lxplus and run:
+
+```bash
+wget https://cern.ch/go/9rKZ && unzip data-shell.zip && rm data-shell.zip
```

Comment:
This won't work until this branch is merged but https://cern.ch/go/9FSJ can be used instead if you want to test it.